jwst/refpix/irs2_subtract_reference.py

```diff
@@ -510,18 +521,20 @@ def replace_refpix(bad_pix, data, bad_mask, is_irs2, low_limit, high_limit,
             v2 = data[:, :, nearest_high, :]
 
     if v1 is not None and v2 is not None:
-        log.debug(f'   Pixel {bad_pix} replaced with value averaged from {nearest_low}, {nearest_high}')
+        log.debug(
+            f"   Pixel {bad_pix} replaced with value averaged from {nearest_low}{{nearest_high}}"
```

Comment:
```suggestion
            f"   Pixel {bad_pix} replaced with value averaged from {nearest_low}, {nearest_high}"
```

Reply:
Not sure why ruff put those extra curly braces in...